x-pack/plugin/rank-rrf/src/main/java/org/elasticsearch/xpack/rank/linear/LinearRankDoc.java

```diff
@@ -30,24 +30,32 @@ public class LinearRankDoc extends RankDoc {
     final float[] weights;
     final String[] normalizers;
     public float[] normalizedScores;
+    public boolean hasValidScore;
 
     public LinearRankDoc(int doc, float score, int shardIndex) {
         super(doc, score, shardIndex);
         this.weights = null;
         this.normalizers = null;
+        this.hasValidScore = false;
```

Comment:
This change doesn't seem to be needed. Can we avoid adding a new transport version just for this.